### PR TITLE
SDK-1140: introduce bootstraptool module

### PR DESCRIPTION
**Pull request by paologalligit**
Introducing new `bootstraptool` module to enable customization when creating a new sidechain

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,73 +1,19 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>io.horizen</groupId>
-	<artifactId>eon</artifactId>
+	<artifactId>EON</artifactId>
```
Comment (paolocappelletti) [inline pom.xml]: lowercase

File: bootstraptool/pom.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.horizen</groupId>
+    <artifactId>bootstraptool</artifactId>
+    <version>0.1.0-SNAPSHOT3</version>
```
Comment (paolocappelletti) [inline bootstraptool/pom.xml]: better use the same snapshot version of the node
next snapshot will  be 0.1.0-SNAPSHOT7

File: node/.run/EON Regtest Node.run.xml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="EON Regtest Node" type="Application" factoryName="Application">
     <option name="MAIN_CLASS_NAME" value="io.horizen.eon.EonApp" />
-    <module name="eon" />
+    <module name="eon5" />
```
Comment (paolocappelletti) [inline node/.run/EON Regtest Node.run.xml]: 5? typo?

File: node/pom.xml [hunk 4]
```diff
@@ -0,0 +1,73 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.horizen</groupId>
+    <artifactId>eon</artifactId>
+    <version>0.1.0-SNAPSHOT3</version>
```
Comment (paolocappelletti) [inline node/pom.xml]: next snapshot will  be 0.1.0-SNAPSHOT7

File: pom.xml (hunk 1 above)
Comment (paolocappelletti) [reply pom.xml]: also check version
next snapshot will  be 0.1.0-SNAPSHOT7

**Review by paolocappelletti — CHANGES_REQUESTED** [review]

**Review by paolocappelletti — APPROVED** [review]

**Comment by paologalligit** [comment]
Closed because replaced by this [PR](https://github.com/HorizenOfficial/eon/pull/16)

**Pull request closed without merging**